src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -563,6 +571,23 @@ public function followRedirect()
         return $response;
     }
 
+    /**
+     * Determines if the response has a meta refresh.
+     *
+     * @return string|null Redirect URI if response has a meta refresh
```

Comment:
Should be removed as it duplicated the signature. The above sentence should be updated instead ("determines" is not accurate anymore.)